lib/phraseapp_updater/phraseapp_api.rb

```diff
@@ -25,10 +25,15 @@ def initialize(api_key, project_id, locale_file_class)
       @locale_file_class = locale_file_class
     end
 
-    def create_project(name, parent_commit)
+    # @param [Hash] opts Options to be passed to the {https://developers.phrase.com/api/#post-/projects PhraseApp API}
```

Comment:
I tried to apply YARD style syntax here but I'm not sure if we have a standard for documenting our ruby code.